nixos/modules/programs/vim.nix

```
@@ -14,7 +14,7 @@ in
 
     defaultEditor = lib.mkEnableOption "vim as the default editor";
 
-    package = lib.mkPackageOption pkgs "vim" { example = "vim-full"; };
+    package = lib.mkPackageOption pkgs "vim" { example = "pkgs.vim-full"; };
```

```suggestion
    package = lib.mkPackageOption pkgs "vim" { example = [ "vim-full" ]; };
```

This avoids duplicating `pkgs`. (This suggestions does not block merge of this PR.)